pkg/services/team/teamimpl/store.go

```diff
@@ -140,6 +140,7 @@ func (ss *xormStore) Delete(ctx context.Context, cmd *team.DeleteTeamCommand) er
 			"DELETE FROM team WHERE org_id=? and id = ?",
 			"DELETE FROM dashboard_acl WHERE org_id=? and team_id = ?",
 			"DELETE FROM team_role WHERE org_id=? and team_id = ?",
+			"DELETE FROM role WHERE org_id=? and name = 'managed:teams:' || ? || ':permissions'",
```

Comment:
Awesome job spotting this! We also need to remove the role's permissions I think :100: 
```suggestion
			"DELETE FROM role WHERE org_id=? and name = 'managed:teams:?:permissions'",
```

Reply:
Thanks @gamab,

Using `"DELETE FROM role WHERE org_id=? and name = 'managed:teams:?:permissions'"` was failing with an error saying: query requires two parameters but found one... That's because the second `?` was ignored. Maybe because it is inside quotes? - Not sure.

But I think its better to change it since Grafana supports multiple databases... Pipes `||` might fail for other databases.

Shall we take it outside that loop and simple prepare the query manually? with `fmt.Sprintf`